torch/csrc/jit/frontend/builtin_functions.cpp

```diff
@@ -72,6 +72,12 @@ def _test_serialization_subcmul_0_2(self: Tensor, other:Tensor, alpha: number=2)
   return other - (self * alpha)
 )SCRIPT";
 
+// Tensor copy creation
+auto tensor_clone = R"SCRIPT(
+def tensor(other:Tensor) -> Tensor:
```

Comment:
I think we also need to handle the other arguments:
https://pytorch.org/docs/stable/torch.html#torch.tensor `torch.tensor(data, dtype=None, device=None, requires_grad=False, pin_memory=False)`.
```
if (dtype is not None) {
    tensor = tensor.to(dtype)
}
...
```

Reply:
my assumption was that tensor.clone() would result in a new tensor with the same properties as the original tensor (e.g. dtype etc.) -- is this not the case? 

are you concerned with copying the dtype from the src tensor, or with providing a convenience wrapper to override it?

oh, i think i get it now- the use case is something like this, i guess?
`@script
def foo(x):
   return torch.tensor(x, dtype=blah)`

Reply:
yep, correct